daemon/protocol/python/tests/daemon_cli_test.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+
+import os
+import re
+import shutil
+import subprocess
+import tempfile
+import time
+import unittest
+
+gopath = os.environ['GOPATH']
+pypath = gopath + '/src/github.com/scootdev/scoot/daemon/protocol/python/scoot'
```

Comment:
We need to handle the case where multiple paths are given. I think the default behavior is to take the first path (where paths are delimited by ':') and drop everything else.